src/runtime/Runtime.Delegates.cs

```diff
@@ -23,7 +23,14 @@ static Delegates()
             Py_EndInterpreter = (delegate* unmanaged[Cdecl]<PyThreadState*, void>)GetFunctionByName(nameof(Py_EndInterpreter), GetUnmanagedDll(_PythonDll));
             PyThreadState_New = (delegate* unmanaged[Cdecl]<PyInterpreterState*, PyThreadState*>)GetFunctionByName(nameof(PyThreadState_New), GetUnmanagedDll(_PythonDll));
             PyThreadState_Get = (delegate* unmanaged[Cdecl]<PyThreadState*>)GetFunctionByName(nameof(PyThreadState_Get), GetUnmanagedDll(_PythonDll));
-            _PyThreadState_UncheckedGet = (delegate* unmanaged[Cdecl]<PyThreadState*>)GetFunctionByName(nameof(_PyThreadState_UncheckedGet), GetUnmanagedDll(_PythonDll));
+            try
+            {
+                _PyThreadState_UncheckedGet = (delegate* unmanaged[Cdecl]<PyThreadState*>)GetFunctionByName(nameof(_PyThreadState_UncheckedGet), GetUnmanagedDll(_PythonDll));
+            }
+            catch (MissingMethodException)
+            {
+                // Not supported in Python 3.13 anymore
```

Comment:
Isn't it that the function merely has been renamed to `PyThreadState_GetUnchecked` (see https://docs.python.org/3/c-api/init.html#c.PyThreadState_GetUnchecked).

Reply:
Great, thanks! I only looked for `PyThreadState_UncheckedGet` :/